resources/com/github/coreos/job.json

```diff
@@ -0,0 +1,52 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "clean-up",
+    "annotations": {
+      "openshift.io/display-name": "Clean Up Job",
+      "description": "Clean Up Job",
+      "template.openshift.io/bindable": "false"
+    }
+  },
+  "objects": [
+  {      
+  "apiVersion": "batch/v1",
+  "kind": "Job",
+  "metadata": {
+    "name": "${NAME}"
+  },
+  "spec": {
+    "ttlSecondsAfterFinished": 10800,
+    "template": {
+      "spec": {
+        "containers": [
+          {
+            "name": "${NAME}",
+            "image": "${IMAGE}",
```

Comment:
Let's just hardcode an image here like Busybox or Fedora and drop the `IMAGE` parameter entirely? Basically anything that has `sleep` in it. :)